### PR TITLE
fixed github comments for sulu-buttons

### PR DESCRIPTION
**Pull request by marcelmoosbrugger**
__Tasks:__
- [x] fix comments of https://github.com/sulu-io/sulu/pull/1421
- [x] behat tests
- [ ] got feedback

__Informations:__

| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| Tests pass?   | none
| Fixed tickets | none
| Doc           | none

### CODE REVIEW

File: bin/build-bundles.sh [hunk 1]
```diff
@@ -23,4 +23,4 @@ for bundle in *; do
         build_bundle
         cd ..
     fi
-done
\ No newline at end of file
+done
```
Comment (wachterjohannes) [inline bin/build-bundles.sh]: empty line at the end of the file

**Comment by wachterjohannes** [comment]
@marcelmoosbrugger add a empty line at the end of `bin/build-bundles` => ACK

**Comment by marcelmoosbrugger** [comment]
@wachterjohannes fixed everything

File: bin/build-bundles.sh (hunk 1 above)
Comment (wachterjohannes) [inline bin/build-bundles.sh]: tock tock (:

**Pull request merged**